runtime/src/bank.rs

```diff
@@ -4676,10 +4676,7 @@ impl Bank {
     /// Process a Transaction. This is used for unit tests and simply calls the vector
     /// Bank::process_transactions method.
     pub fn process_transaction(&self, tx: &Transaction) -> Result<()> {
-        self.try_process_transactions(std::iter::once(tx))?[0].clone()?;
```

Comment:
Out of scope for this PR, but given the "This is used for unit tests and ..." comment, this should seemingly be DCOU